com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -30,13 +34,30 @@ public class NetworkConfig
         [Tooltip("When set, NetworkManager will automatically create and spawn the assigned player prefab. This can be overridden by adding it to the NetworkPrefabs list and selecting override.")]
         public GameObject PlayerPrefab;
 
+#if NETCODE_USE_ADDRESSABLES
+        /// <summary>
+        /// The default player prefab addressable
+        /// </summary>
+        [Tooltip("When set, NetworkManager will automatically create and spawn the assigned player prefab. This can be overridden by adding it to the NetworkPrefabs list and selecting override.")]
+        public AssetReferenceGameObject PlayerAddressable;
```

Comment:
Note to myself... I forgot to handle this. I need to add code to load this and set PlayerPrefab, and a test to make sure it's working.